fix_encoder_lint.sh

```
@@ -0,0 +1,242 @@
+cat << 'INNER_EOF' > src/main/webapp/WEB-INF/jsp/messenger/ViewAttachment.jsp.patch
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)